packages/fileeditor-extension/src/index.ts

```diff
@@ -205,6 +207,27 @@ function activate(
     updateWidget(widget.content);
   });
 
+  // Add a command to change font size.
+  commands.addCommand(CommandIDs.changeFontSize, {
+    execute: args => {
+      const delta = args['delta'] as number;
```

Comment:
I think this can be a touch more defensive in case the args are not as expected:
```typescript
const delta = args['delta'] as number || 0;
```

Reply:
I'm not an expert on TypeScript and therefore I can be wrong, but I think the row
```
const delta = args['delta'] as number || 0;
```
is transformed to
```
var delta = args['delta'] || 0;
```
Provided that `args['delta']`is evaluated to `true`, `delta` value is `args['delta']` even if it is not a number.
For example if `args['delta']` coincides with the string `'test'`, then `delta` value is the string `'test'`not the number `0`.
So this isn't much more defensive.

Reply:
In fact it catches only `undefined` cases, `null`inside arithmetic operation is already transformed in `0`. Maybe something like

```
const delta = Number(args['delta']);
if (Number.isNaN(delta))
  return;
```

is more robust. What do you think?

Reply:
Sounds good to me.